server/filter.js

```diff
@@ -1,97 +1,82 @@
 "use strict";
 const remote = require("@electron/remote");
+const { debounce } = require("./utils");
+var filterQuery = "",
+    filterValidEvents = false;
 
-var filterTimer,
-    filterState = false;
+var update = debounce(filterItems, 50);
 
-function filterItems(items, value) {
-    if (typeof value === "undefined") {
-        value = document.getElementById("filter-events").value;
-    }
-    if (value === "") {
-        clearFilter();
-    }
-    // we're filtering case insensitive
-    value = value.toLowerCase();
+function filterItems() {
```

Comment:
I am getting this error when looking at the code.
Should `filterItems()`  `getSchemaName()` and `contains()` be moved to the top?

![Screenshot 2022-12-20 at 09 40 22](https://user-images.githubusercontent.com/14923445/208622137-2a1b5493-a9c3-4276-b880-4005e6471bfe.png)